internal/resource/default_user_secret_test.go

```diff
@@ -100,6 +103,25 @@ var _ = Describe("DefaultUserSecret", func() {
 				Expect(port).To(BeEquivalentTo("5672"))
 			})
 
+			By("Setting a connection string", func() {
+				connection_string, ok = secret.Data["connection_string"]
+				Expect(ok).To(BeTrue(), "Failed to find a key \"connection_string\" in the generated Secret")
+
+				username, ok = secret.Data["username"]
+				Expect(ok).To(BeTrue(), "Failed to find a key \"username\" in the generated Secret")
+
+				password, ok = secret.Data["password"]
+				Expect(ok).To(BeTrue(), "Failed to find a key \"password\" in the generated Secret")
+
+				host, ok = secret.Data["host"]
+				Expect(ok).To(BeTrue(), "Failed to find a key \"host\" in the generated Secret")
+
+				port, ok = secret.Data["port"]
+				Expect(ok).To(BeTrue(), "Failed to find a key \"port\" in the generated Secret")
```

Comment:
This works, but it would read better by using the matcher `HaveKey` ([link](https://onsi.github.io/gomega/#havekeykey-interface)).

Reply:
Great suggestion! I am not that familiar with go and the framework tests so I didn't know the most optimal way and I copied.
Did what you suggested, can you take another look?